### PR TITLE
Update Automation UI parser [RN35435] [DCP198609] [DCP171211] [DCP118234]

### PR DESCRIPTION
**Pull request by HansSolos**
References: [RN35435] [DCP198609] [DCP171211] [DCP118234]

### CODE REVIEW

File: src/DataMiner/SLManagedAutomation/UIBuilder.cs [hunk 1]
```diff
@@ -74,35 +74,59 @@ public class UIBuilder
...
+        /// Gets or sets the column with the highest number that has one or more components.
+        /// </summary>
+        /// <value>The column with the highest number that has one or more components.</value>
+        /// <example>
+        /// <code>
+        /// UIBuilder uib = new UIBuilder();
+        /// uib.MaxCol = 2;
+        /// </code>
+        /// </example>
+        public int MaxCol { get; set; }
```
Comment (Skyline-ThomasGH) [inline src/DataMiner/SLManagedAutomation/UIBuilder.cs:87]: Can you explain the purpose of this property?

Comment (HansSolos) [reply src/DataMiner/SLManagedAutomation/UIBuilder.cs:87]: the purpose of this property is to get or set the column with the highest number that has one or more components.

File: src/DataMiner/SLNetTypes/Automation/AutomationTimeUpDownOptions.cs [hunk 2]
```diff
@@ -71,14 +71,20 @@ public class AutomationTimeUpDownOptions : AutomationConfigOptions
...
-		/// <value>The number of digits to be used in order to represent the fractions of seconds.</value>
-		/// <remarks>
-		/// <para>Default: 0.</para>
-		/// </remarks>
-		public int FractionalSecondsDigitsCount { get; set; }
+        /// <summary>
+        /// Gets the default value for the <see cref="AutomationTimeUpDownOptions.ShowTimeUnits"/> property.
+        /// </summary>
+        /// <value>The default value for the <see cref="AutomationTimeUpDownOptions.ShowTimeUnits"/> property.</value>
+        public static Boolean DEFAULT_SHOW_TIMEUNITS { get; protected set; }
```
Comment (Pieter-JanV) [inline src/DataMiner/SLNetTypes/Automation/AutomationTimeUpDownOptions.cs:78]: The type is not consistent with the others (e.g. see DEFAULT_SHOW_SECONDS). It however matches with what is defined in the related server commit.

**Comment by EdithLansens** [comment]
@PedroDebevere OK to merge?

**Review by PedroDebevere — APPROVED** [review]

**Pull request merged**